project-fob/Controllers/HostController.cs

```diff
@@ -12,6 +12,12 @@ namespace project_fob.Controllers
 {
     public class HostController : Controller
     {
+        public static void Foo()
+        {
+            System.Console.WriteLine("Foo");
+        }
+
+
```

Comment:
Some left over testing code here (I know I wrote it 😄, but we should still kill it before merging into `master`)